detectors/pcl_detector/params/example_params.yaml

```diff
@@ -0,0 +1,23 @@
+pcl_detector:
+  leaf_size: 0.05
+  detector_type: "dbscan" # one of "dbscan", "euclidean", "gmm", "optics"
```

Comment:
maybe remove "gmm" and "optics" from comments since the gmm and optics files are deleted